proto/decentraland/sdk/components/avatar_customization.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package decentraland.sdk.components;
+import "decentraland/common/colors.proto";
+import "decentraland/sdk/components/common/id.proto";
+option (common.ecs_component_id) = 1087;
+
+// AvatarCustomizations sets all modifiers over the avatar's apparence. This 
+message PBAvatarCustomization {
```

Comment:
The word `Customization` is a little long. How about `Mods`, for modifiers? I'm not sure, nothing is better is coming to mind right now.